### DISCUSSION TITLE
Which letter is missing in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1146] [Galaxy Trivia] [answered]
```
B C E G K Q S W
```

**Comment by thuongtruong1009** [comment]
The missing letter in the sequence is "M." 

The sequence appears to be doubling the letters in the alphabet: B, C, E, G, K, Q, S, W. So, after "W," the next letter would be "M."

**Comment by tomerf** [comment, marked as answer]
The letters are representing the prime numbers:
B = 2
C = 3
E = 5
G = 7
K = 11
Q = 17
S = 19
W = 23
So the missing letter is M = 13 which is also prime but missing from the sequence.

**Reply by ParanoidUser** [reply]
Well done! 👏👏
